pkg/operation/seed/seed.go

```diff
@@ -808,8 +815,14 @@ func RunReconcileSeedFlow(
 			"networkPolicy": map[string]interface{}{
 				"additionalEgressIPBlocks": additionalEgressIPBlocks,
 			},
+			"eventLogger": map[string]interface{}{
+				"enabled": eventLoggingEnabled,
+			},
+		},
+		"loki": lokiValues,
+		"event-logger": map[string]interface{}{
+			"enabled": false,
```

Comment:
Shouldn't this be:
```suggestion
			"enabled": eventLoggingEnabled,
```
If the value is used at all.